utils/cryptutils/cryptutils.go

```diff
@@ -408,11 +408,8 @@ func SavePrivateKeyToFile(fileName string, key crypto.PrivateKey) error {
 
 	switch privateKey := key.(type) {
 	case *rsa.PrivateKey:
-		if err = pem.Encode(file, &pem.Block{
-			Type:  PEMBlockRSAPrivateKey,
-			Bytes: x509.MarshalPKCS1PrivateKey(privateKey),
-		}); err != nil {
-			return aoserrors.Wrap(err)
+		if err = PrivateKeyToPEM(file, privateKey); err != nil {
```

Comment:
Then no need to cast private key type here. Use PrivateKeyToPEM for all keys.

Reply:
Done